docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -6,7 +6,7 @@
 
 This topic assumes that you know about Persistence in Hazelcast. To learn about Persistence, see the xref:hazelcast:storage:persistence.adoc[Platform documentation].
 
-There are two options for a backup: local and external. Local backups are kept in volume and never moved anywhere whereas external backups are moved into buckets provided by the user. 
+There are two options for a backup: local and external. Local backups are kept in volume and never moved anywhere whereas external backups are moved into buckets provided by the user.
```

Comment:
```suggestion
There are two options for a backup: local and external. Local backups are kept in volume and never moved. External backups are moved into buckets provided by the user.
```

Reply:
or a list, something like:

Backups can be either of the following:

- Local. Local backups are....
- External. External backups are...

Reply:
https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/171/commits/29e7229d15fce92bb46b030a72d16010c16fecd7